src/main/scala/Compiler.scala

```diff
@@ -16,15 +16,17 @@ object Compiler {
     "Hoist memory reads" -> passes.HoistMemoryReads,
     "Sequentialize" -> passes.Sequentialize,
     "Lower unroll and bank" -> passes.LowerUnroll,
-    "Lower for loops" -> passes.LowerForLoops
+    "Lower for loops" -> passes.LowerForLoops,
+    "Hoist slow binops" -> passes.HoistSlowBinop
   )
 
   // Transformers to execute *after* type checking. Boolean indicates if the
   // pass should only run during lowering.
-  val postTransformers: List[(String, (TypedPartialTransformer, Boolean))] = List(
-    "Rewrite views" -> (passes.RewriteView, false),
-    "Add bitwidth" -> (passes.AddBitWidth, true)
-  )
+  val postTransformers: List[(String, (TypedPartialTransformer, Boolean))] =
+    List(
+      "Rewrite views" -> (passes.RewriteView, false),
+      "Add bitwidth" -> (passes.AddBitWidth, true)
+    )
```

Comment:
When submitting PRs, aspire to minimize the diffs so that they don't show unnecessary changes. If you're going to reformat a whole file, commit the formatting changes separately (this is standard practice at every software company)